lib/actions/os.coffee

```diff
@@ -16,25 +16,76 @@ limitations under the License.
 
 commandOptions = require('./command-options')
 
+formatVersion = (v, isRecommended) ->
+	result = "v#{v}"
+	if isRecommended
+		result += ' (recommended)'
+	return result
+
+resolveVersion = (deviceType, version) ->
+	if version isnt 'menu'
+		return Promise.resolve(version)
+
+	form = require('resin-cli-form')
+	resin = require('resin-sdk-preconfigured')
```

Comment:
For performance reasons the CLI lazy-loads the dependencies. node caches them anyway, but they're only requested when first needed. Page and Juan has implemented this change to aid the specific performance issues.